src/components/BasicMap/BasicMap.js

```diff
@@ -49,6 +49,9 @@ const propTypes = {
    */
   onFeaturesClick: PropTypes.func,
 
+  /** Hit-detection tolerance in pixels. Pixels inside the radius around the given position will be checked for features. */
+  featuresClickHitTolerance: PropTypes.number,
```

Comment:
I guess we could want to pass other options. call it  featuresClickOptions then pass the all object to getFeaturesAtPixel call.

Reply:
great idea. I will change that

Reply:
https://github.com/candux/react-spatial/pull/1